build.assets/build-package.sh

```diff
@@ -80,6 +80,12 @@ if [ ! $(command -v curl) ]; then
     exit 2
 fi
 
+# check that tar is installed
+if [ ! $(command -v tar) ]; then
```

Comment:
hmm, on my system `command` doesn't have a `-v` flag.
is this portable? it's possible that my setup is just too weird.

Reply:
Looks like it does exist on `bash` and `zsh` - I'm unsure whether it's a generally supported thing. It's effectively the same as `which tar` but built into the shell. What shell are you using?

Reply:
using `fish`, where the flag is `command -s` to search the `$PATH`
can we assume that `which` is more universal?

Reply:
Here's a good answer which goes into a lot of detail: https://unix.stackexchange.com/a/85250

Does `fish` have `type`? The answer seems to roughly indicate that `type` > `command -v` > `which`, so I'd prefer to go better (by using a built-in shell command like `type`) rather than 'worse' (by using `which`)

Reply:
Yeah, there's `type`:
```
> type tar
tar is /bin/tar
```